requirements/_base.txt

```diff
@@ -6,59 +6,64 @@
 #
 aiodebug==2.3.0
     # via simcore-service-library
-aiofiles==22.1.0
+aiofiles==23.1.0
     # via simcore-service-library
-aiohttp==3.8.3
+aiohttp==3.8.4
     # via
     #   -r requirements/_base.in
     #   aiozipkin
     #   simcore-service-library
 aiopg==1.4.0
     # via simcore-service-library
-aiosignal==1.2.0
+aiosignal==1.3.1
     # via aiohttp
 aiozipkin==1.1.1
     # via simcore-service-library
 async-timeout==4.0.2
     # via
     #   aiohttp
     #   aiopg
-attrs==22.1.0
+attrs==22.2.0
     # via
     #   aiohttp
     #   jsonschema
     #   openapi-core
+    #   pytest
     #   simcore-service-library
-certifi==2022.9.24
+certifi==2022.12.7
     # via requests
-charset-normalizer==2.1.1
+charset-normalizer==3.0.1
```

Comment:
I guess this is not a problem since you are running tests, but it looks like you upgraded the full set of requirements when you added pytest-simcore in the requirements.
Also, I think pytest-simcore is only used for testing right? so it should go in _test.in and not in _base.in.

pytest-simcore is not needed for normal usage right?

Also when you add pytest-simcore in _test.in you can also selectively upgrade the requirements
```
cd requirements
make touch
make reqs upgrade=pytest-simcore
```

Reply:
Agreed: selective upgrade is important.

Reply:
noted for the future, for now it will be like this I'm afraid :) But your point is absolutely valid! thanks